guides/source/active_job_basics.md

```diff
@@ -10,6 +10,7 @@ After reading this guide, you will know:
 
 * How to create jobs.
 * How to enqueue jobs.
+* How to pass arguments to a jobs.
```

Comment:
:scissors: ; there's no need to list all the things we cover in the guide, just the main points!